packages/mui-material/src/ButtonGroup/ButtonGroupButtonContext.ts

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+
+interface ButtonGroupButtonContextType {
+  firstButton?: boolean;
+  lastButton?: boolean;
```

Comment:
I'm not 100% sure about these names. What do you think about adding the `is` prefix?

```suggestion
  isFirstButton?: boolean;
  isLastButton?: boolean;
```

or just `isFirst`/`isLast`?

Reply:
In my experience I see folks naming boolean variables without prefixes (An article I found https://javascript.plainenglish.io/naming-boolean-variables-prefixes-df973d7d7ec3#:~:text=Prefixes%20are%20nice%2C%20but%20not,thing%20you%20care%20about%20most.). But what's important is consistency and convention in the codebase. So, what do we use generally? 

Also I'd go with having `button` in the name because the context is used inside the Button component only.

Reply:
Alright, I have renamed to use the `is` prefix.